fenix/app/src/test/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckNavigationMiddlewareTest.kt

```diff
@@ -1,75 +1,57 @@
 package org.mozilla.fenix.shopping.middleware
 
+import io.mockk.MockKAnnotations
+import io.mockk.impl.annotations.MockK
+import io.mockk.verify
+import mozilla.components.feature.tabs.TabsUseCases
 import mozilla.components.support.test.ext.joinBlocking
 import mozilla.components.support.test.libstate.ext.waitUntilIdle
+import mozilla.components.support.test.robolectric.testContext
 import mozilla.components.support.test.rule.MainCoroutineRule
-import org.junit.Assert.assertEquals
+import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.helpers.FenixRobolectricTestRunner
 import org.mozilla.fenix.shopping.store.ReviewQualityCheckAction
-import org.mozilla.fenix.shopping.store.ReviewQualityCheckState
 import org.mozilla.fenix.shopping.store.ReviewQualityCheckStore
 
+@RunWith(FenixRobolectricTestRunner::class)
 class ReviewQualityCheckNavigationMiddlewareTest {
 
     @get:Rule
     val coroutinesTestRule = MainCoroutineRule()
     private val dispatcher = coroutinesTestRule.testDispatcher
     private val scope = coroutinesTestRule.scope
+    private lateinit var store: ReviewQualityCheckStore
 
-    @Test
-    fun `WHEN opening an external link THEN the link should be opened in a new tab`() {
-        var isOpenedInSelectedTab = false
-        var isOpenedInNewTab = false
-        val store = ReviewQualityCheckStore(
+    @MockK(relaxed = true)
```

Comment:
❓ Do we need to mock this? We could use the real object using a `BrowserStore` object and assert the store state that the url is opened in a tab.